source/manual/post-a-statuspage-message.html.md

```diff
@@ -7,22 +7,29 @@ parent: "/manual.html"
 important: true
 ---
 
-When something goes wrong on GOV.UK we follow our [incident management procedure][inc].
-As part of that we update our status page [status.publishing.service.gov.uk][status].
+When something goes wrong on GOV.UK, we follow our [incident management procedure][inc].
+As part of that, we update our status page [status.publishing.service.gov.uk][status].
 
 The page is managed using a third party service called [Statuspage][sp].
+This is what the [Statuspage admin interface][man] looks like:
```

Comment:
I'd remove this section - admin interfaces have a habit of changing, and I'm not sure a screenshot here adds anything.

Reply:
Thanks, I thought the same tbh.